src/ZipkinOpenTracing/Tracer.php

```diff
@@ -91,6 +86,11 @@ public function startSpan($operationName, $options = [])
             $options = StartSpanOptions::create($options);
         }
 
+        if (!$this->hasParentInOptions($options) && $this->getActiveSpan() !== null) {
```

Comment:
Does this change should be reflect in tests?

Reply:
Any ideas on how this could be tested?

One thing comes to my mind is that in a test we can check if `newTrace` was not called but I cannot prophesize `ZipkinOpenTracing\Tracer` as it is marked as `final`